.NET/Microsoft.Recognizers.Text.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2026
+# Visual Studio Version 16
```

Comment:
Please be careful to not modify the solution/project files unless intentonal.

Reply:
sorry pushed by mistake, reverted now.